### DISCUSSION TITLE
Support on GoCD

### DISCUSSION THREAD

**Discussion by GaelEtatGeneve** [gocd/gocd#10268] [Q&A]
Hello, we have set up an industrialized deployment platform based on GoCD.
 
As such, I am looking for a company that could provide us with support for about twenty days annually on:
- Version upgrades
- Expertise
- Adding a module/plugin/apis
- Development of new features
 
Do you know a company that offers this type of service?

Kind regars,

Gael

**Comment by chadlwilson** [comment]
Unfortunately I'm not personally aware of any company that is providing this kind of paid support since the project become fully "community driven" open source at the end of 2020, upon which Thoughtworks stopped providing commercial support. Personally I have tried to step in during the last 9 months to try and 'keep the lights on' regarding security patching, dependency updates, replacement of some deprecated/EOL libraries, and generally trying to improve the dev experience a bit. After 22.1.0, I hope to refocus around bug fixes and minor improvements as I build more context, however I do have limited spare time outside my day job.

There is also not currently a 'proper' way to financially support the ongoing development or maintenance of GoCD, or the ongoing maintenance of its infrastructure (incl. the website, AWS build infra, TLS certs etc).

I have had a couple of offline/private discussions with folks who have at least contemplated providing this kind of support, but not aware of any decisions on that. Commercial viability is likely to be a concern.

Nevertheless, I'd encourage your folks to engage with the community either here or on the Google Groups or Gitter to get some help, understand the impact of upgrades and ask questions.
